src/Native/CMakeLists.txt

```diff
@@ -132,7 +132,9 @@ if (CLR_CMAKE_PLATFORM_UNIX)
         # We cannot enable "stack-protector-strong" on OS X due to a bug in clang compiler (current version 7.0.2)
         add_compile_options(-fstack-protector)
     else()
-        add_compile_options(-fstack-protector-strong)
+        if(NOT CLR_CMAKE_PLATFORM_WASM)
```

Comment:
Nit: this can be folded into the `else` above with `elseif`

Reply:
Done